driver/src/driver/stablex_driver.rs

```diff
@@ -25,17 +27,28 @@ impl<'a> StableXDriver<'a> {
         // Try to process previous batch auction
         let batch = self.contract.get_current_auction_index()? - 1;
         if self.past_auctions.contains(&batch) {
-            info!("Already processed batch {}", batch);
             return Ok(false);
         }
         let (account_state, orders) = self.contract.get_auction_data(batch)?;
-        let solution = self.price_finder.find_prices(&orders, &account_state)?;
+        let solution = if orders.is_empty() {
+            info!("No orders in batch {}", batch);
+            Solution::trivial(0)
+        } else {
+            let solution = self.price_finder.find_prices(&orders, &account_state)?;
+            info!("Computed solution: {:?}", &solution);
+            solution
+        };
 
-        info!("Submitting solution: {:?}", &solution);
-        self.contract.submit_solution(batch, orders, solution)?;
+        let submitted = if solution.executed_sell_amounts.iter().sum::<u128>() > 0 {
+            self.contract.submit_solution(batch, orders, solution)?;
+            info!("Successfully applied solution to batch {}", batch);
+            true
+        } else {
+            info!("Not submitting 0 volume solution for batch {}", batch);
```

Comment:
Should (or could) this say something like "not submitting trivial solution"

Is a zero volume solution equivalent to trivial? I imagine it must be.